tests/pkcs/pkcs8-test.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+if [ ! -d ./certs/ ]; then
+    #return 77 to indicate to automake that the test was skipped
+    exit 77
+fi
+
+# Skip test if filesystem disabled
+FILESYSTEM=`cat config.log | grep "disable\-filesystem"`
+if [ "$FILESYSTEM" != "" ]
+then
+    exit 77
+fi
+
+RESULT=`./wolfssl pkcs8 -in certs/server-keyEnc.pem -passin pass:yassl123 2>&1`
+echo "$RESULT" | grep "Recompile wolfSSL with PKCS8 support"
+if [ $? == 0 ]; then
+    #return 77 to indicate to automake that the test was skipped
+    exit 77
+fi
+
+run() {
+    RESULT=`./wolfssl $1`
+
+    if [ $? != 0 ]; then
+        echo "Failed on test \"./wolfssl $1\""
+        exit 99
+    fi
+}
+
+run "pkcs8 -in certs/server-keyEnc.pem -passin pass:yassl123 -outform DER -out keyEnc.der"
```

Comment:
Let's add a couple of negative tests here. Like using server-cert.pem as an input and making sure it gracefully fails and propagates the fail return value. This so far only tests valid inputs.

Reply:
Added a couple negative tests